templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -1,61 +1,31 @@
 {% extends 'base.html.twig' %}
-
 {% block stylesheets %}
-    {{ encore_entry_link_tags('exception') }}
+    {{ encore_entry_link_tags('web-site') }}
 {% endblock %}
-
 {% block body %}
-
-    <div class="container mt-5">
-    <div class="row">
-        <div class="col-8 offset-2 display-1 text-center">
-
-            <!-- line 01 -->
-            <div class="row text-danger">
-
-                <div class="col-2">
-                    <div class="border border-danger rounded">E</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">O</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">!</div>
-                </div>
-
-            </div>
-
-            <div class="row">
-                <div class="col-12 text-danger"><i class="fa fa-warning"></i></div>
-                <div class="col-12 text-center text-uppercase  h1 text-dark">Internal Server Error</div>
-            </div>
-
-            <div class="row">
-                {% if is_granted('IS_AUTHENTICATED_FULLY') %}
-                <div class="col-6">
-                    {% else %}
-                    <div class="col-6 offset-3">
-                        {% endif %}
-                        <a href="{{ path('app_homepage') }}" class="btn btn-danger btn-block"><i class="fa fa-home "></i> Return to the homepage </a>
-                    </div>
-
-                    {% if is_granted('IS_AUTHENTICATED_FULLY') %}
-                        <div class="col-6">
-                            <a href="{{ path('app_profile', {'slug': app.user.slug}) }}" class="btn btn-primary btn-block"><i class="fa fa-user-circle"></i> My profile </a>
-                        </div>
-                    {% endif %}
-                </div>
-
-            </div>
-        </div>
+    <div class="container mt-5 has-text-centered">
+        <h1 class="is-size-4">{{ message| default('Internal server error') }}</h1>
```

Comment:
@ibonkonesa I guess bulma there's not any class `is-size-4`

Reply:
Bulma uses this class:

https://bulma.io/documentation/helpers/typography-helpers/